modules/installation-configuration-parameters.adoc

```diff
@@ -1000,7 +1000,11 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Number
 
 |`<machine-pool>.platform.ovirt.vmType`
-|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.
+|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.  By default, master nodes use `high-performance`, and worker nodes use `server`. For details, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Virtual_Machine_General_settings_explained[Explanation of Settings in the New Virtual Machine and Edit Virtual Machine Windows] in the _Virtual Machine Management Guide_.
```

Comment:
Why not link https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools ?

Reply:
Done.